app/build.gradle.kts

```diff
@@ -164,7 +143,7 @@ dependencies {
     kaptAndroidTest(libs.hilt.android.compiler)
     ksp(libs.androidx.room.compiler)
     ksp(libs.compose.destinations.ksp)
-    lintChecks(project(":lint-checks"))
+//    lintChecks(project(":lint-checks"))
```

Comment:
Need to add lint checks back in before merging, or right after. Biggest issue is defining the version numbers right in the lint-checks/build.gradle file.